Incompatible Type Engine when using Version 6 in TypeScript project

I updated to the latest version of express-handlebars in my TypeScript Project and am now getting the following error:
```
error TS2345: Argument of type 'Engine' is not assignable to parameter of type '(path: string, options: object, callback: (e: any, rendered?: string | undefined) => void) => void'.
  Types of parameters 'options' and 'options' are incompatible.
    Type 'object' is not assignable to type 'EngineOptions'.
      Index signature for type 'string' is missing in type '{}'.
```

for the following code:
```ts
app.engine('hbs', engine({
    extname: '.hbs',
    defaultLayout: 'default',
    helpers: {
        ...helpers
    }
}));
```

I use the latest TypeScript (`4.5.2`) and have strict mode enabled if that matters.

--- a/spec/express-handlebars.test.ts
+++ b/spec/express-handlebars.test.ts
@@ -2,6 +2,7 @@ import * as path from "path";
 import * as expressHandlebars from "../lib/index";
 import type {
 	TemplateDelegateObject,
+	EngineOptions,
 } from "../types";
 
 function fixturePath (filePath = "") {
@@ -365,7 +366,7 @@ describe("express-handlebars", () => {
 		test("should render html", async () => {
 			const renderView = expressHandlebars.engine({ defaultLayout: null });
 			const viewPath = fixturePath("render-text.handlebars");
-			const html = await renderView(viewPath, { text: "test text" });
+			const html = await renderView(viewPath, { text: "test text" } as EngineOptions);
 			expect(html).toBe("<p>test text</p>");
 		});
 	});

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,3 +1,5 @@
+/// <reference types="handlebars" />
+
 export interface UnknownObject {
 	[index: string]: unknown
 }
@@ -62,7 +64,7 @@ export type HandlebarsCompile = (input: unknown, options: CompileOptions) => Han
 export type HandlebarsPrecompile = (input: unknown, options: PrecompileOptions) => TemplateSpecification;
 
 export interface HandlebarsImport {
-	[index: string]: any;
+	[index: string]: unknown;
 	compile: HandlebarsCompile;
 	precompile: HandlebarsPrecompile;
 }
@@ -87,4 +89,4 @@ export interface RenderCallback {
 		(err: Error|null, content?: string): void;
 }
 
-export type Engine = (viewPath: string, options: EngineOptions, callback?: RenderCallback) => Promise<string>
+export type Engine = (viewPath: string, options: ConfigOptions, callback?: RenderCallback) => Promise<string>

